### DISCUSSION TITLE
Calibration Error metric issue

### DISCUSSION THREAD

**Discussion by abhisheksambyal** [Lightning-AI/torchmetrics#430] [Classification] [answered]
I am using Torchmetrics v0.4.1.
When I try to import **CalibrationError** metric it gives me this error **AttributeError: module 'torchmetrics' has no attribute 'CalibrationError'**.

Please help.

**Comment by SkafteNicki** [comment, marked as answer]
Hi @abhisheksambyal,
The new CalibrationError metric will be released in version 0.5 of torchmetrics, which should release tomorrow.
If you cannot wait you can always install from master to get the latest changes:
```bash
pip install git+https://github.com/PytorchLightning/metrics.git@master
```

**Reply by abhisheksambyal** [reply]
@SkafteNicki Thanks!
